### PR TITLE
TestRun: layout

### PR DESCRIPTION
**Pull request by rwaldron**
image

### CODE REVIEW

File: client/components/TestRun/index.jsx [hunk 1]
```diff
@@ -231,47 +238,69 @@ class TestRun extends Component {
...
+                            src={`/aria-at/${git_hash}/${
+                                tests[this.state.currentTestIndex - 1].file
+                            }?at=${at_key}`}
+                            id="test-iframe"
+                            ref={this.testIframe}
+                        ></iframe>
+                    );
+
+                    menuUnderIframe = (
+                        <ButtonToolbar className="testrun__button-toolbar--margin">
```
Comment (spectranaut) [inline client/components/TestRun/index.jsx:272]: I wonder if we should give these some kind of landmark region or at least an accessible name to the group.

Make just "region" -- the point being, I want it to be easy for a screen reader user to know those buttons are there and skip to them if partially thru a test..? Regions are all about skipping to area of the web page. I don't know what to label the two different button groups -- or if we make them into "regions", what to call the regions.

File: client/components/TestRun/index.jsx [hunk 2]
```diff
@@ -295,41 +324,54 @@ class TestRun extends Component {
...
-                <Helmet>
-                    <title>{`Testing ${apg_example_name} for ${at_name} ${at_version} with ${browser_name} ${browser_version} | ARIA-AT`}</title>
-                </Helmet>
-                {heading}
-                {content}
+            <Fragment>
+                <Container fluid>
+                    <Row>
+                        <Col>
+                            <Helmet>
```
Comment (spectranaut) [inline client/components/TestRun/index.jsx:350]: I think the Helmet doesn't have to be in a row and col, it's just for changing the title element of the webpage.

File: client/components/TestRun/index.jsx (hunk 1 above)
Comment (zcorpan) [reply client/components/TestRun/index.jsx:272]: I'd need to see the generated HTML (or DOM) of the whole page to evaluate this. But a few buttons doesn't necessarily require a landmark.

Comment (spectranaut) [reply client/components/TestRun/index.jsx:272]: ok ignore this concern @rwaldron

**Pull request merged**